src/TourPortal.js

```diff
@@ -187,7 +184,7 @@ class TourPortal extends Component {
         prevState => {
           if (prevState.observer) {
             setTimeout(() => {
-              prevState.observer.disconnect()
+              //prevState.observer.disconnect()
```

Comment:
We don't need to `disconect()` any more?

Reply:
My bad!